src/integration/common/canonical/a11y.js

```diff
@@ -0,0 +1,16 @@
+import '@testing-library/jest-dom/extend-expect';
+import { renderAsReact } from '../../render';
+
+export default ({ pageUrl }) => {
+  describe('a11y', () => {
+    beforeEach(() => renderAsReact(pageUrl));
+
+    it('should render an H1 with the correct attributes', () => {
+      const headlineEl = document.querySelector(
```

Comment:
Why can we rely on document being implicitly in scope for this test but not for `src/integration/common/canonical/SEO.js`?

I think it's because we are using `renderAsReact` rather than `renderAsDom` is that right? We may want to call out this naunce in the docs for the integration tests.

Reply:
That would be the best approach:
https://github.com/bbc/simorgh/pull/5764/files#diff-a5fda338132fd8f79655e950f016fe5aR133

Putting document in global scope for both apis would reduce confusion 🙂